lib/modules/manager/helmv3/artifacts.ts

```diff
@@ -46,12 +46,14 @@ async function helmCommands(
     });
 
   // if credentials for the registry have been found, log into it
-  registries.forEach((value) => {
-    const loginCmd = generateLoginCmd(value, 'helm registry login');
-    if (loginCmd) {
-      cmd.push(loginCmd);
-    }
-  });
+  await Promise.all(
```

Comment:
use pMap from utils

Reply:
Why a map at all 🤔 
This should work with p-all too.